src/config.json

```diff
@@ -1,4 +1,4 @@
 {
-  "django_server": "https://webblog1234.herokuapp.com",
+  "django_server": "http://127.0.0.1:8000",
```

Comment:
Do we need to push localhost URL on github?

Reply:
The frontend uses this variable for API calls. It is the url for backend API.